bot_box/feature_selector.py

```diff
@@ -41,6 +41,14 @@ def get_feature(user_input=""):
             return None
         return bot_info.bot_info_api(query)
 
+    elif user_input.startswith("joke"): #TODO: Find better filter for it. 'joke' conflicts with 'joke me' but should as of now
```

Comment:
I will not conflict with `joke me` as if the command contains 'joke me' then it will be returned earlier and will not be compared to this line.